contentVersionRelatedListLWC/Readme.md

```diff
@@ -0,0 +1,21 @@
+# Lightning Web Component for Content Version List
+
+This component was built to show the list of content versions for the content document object.
+ ![Content Version Related List sample](contentversion.PNG)
+
+## Installation Instructions
+
+1. Deploy the package in the respective org.

```

Comment:
@rajkrishnasamy since this is an LWC, I assume it needs to be deployed via sfdx? Could you verify and if so, amend the readme?

Reply:
@mvogelgesang We need sfdx only to create LWC. Once created we can deploy the package using ant as we do today. The deployment via ant was tested in sandboxes.